### PR TITLE
JJDZR12IL-2 - Add maven project

### PR DESCRIPTION
**Pull request by olszewskiszym**

### CODE REVIEW

**Comment by Ne0rin** [comment]
Its ok

**Review by adrianna-gawron — APPROVED** [review]
Ok

**Review by Ne0rin — APPROVED** [review]
its ok

File: InfiniteLoopers-App/src/main/java/com/isa/App.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.isa;
+
+/**
+ * Hello world!
+ *
```
Comment (malik-maciej) [inline InfiniteLoopers-App/src/main/java/com/isa/App.java:5]: zbędny komentarz

**Review by malik-maciej — APPROVED** [review]

**Pull request merged**